guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -40,33 +40,38 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
     var config = {
         restrict    : 'E',
         replace     : true,
+        scope: {
+            /**
+             * When isActive is set to true then the Clipboard data will be
+             * displayed in the Clipboard Editor. When false, the Clipboard Editor
+             * will not be updated with Clipboard data.
+             * 
+             * @type Boolean
+             */
+            isActive : '=?',
```

Comment:
I don't think you need this? This `scope` in the directive config is used for passing arguments to the directive - which you're not doing that I can see. You can just set stuff directly on `$scope` in the controller.

Reply:
I was originally intending for it to be optional so that it could be set to true for any reason. This doesn't really make much sense so I removed it